build.sbt

```diff
@@ -8,7 +8,7 @@ val circeVersion = "0.13.0"
 val sttpVersion = "3.3.15"
 val Specs2Version = "4.6.0"
 val artifactory = "https://cognite.jfrog.io/cognite/"
-val cogniteSdkVersion = "1.5.18"
+val cogniteSdkVersion = "1.5.19-SNAPSHOT"
```

Comment:
TODO Update this once https://github.com/cognitedata/cognite-sdk-scala/pull/413 is merged